lib/etc/openturns.conf.in

```diff
@@ -434,6 +437,8 @@
 
   <!-- OT::DomainImplementation parameters -->
   <Domain-SmallVolume value_float="1.0e-12" />
+  <Domain-DefaultInColor value_str="#FF7F0E" />
```

Comment:
Introduce a new color code in "DrawableImplementatio.cxx" instead

Reply:
done